src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -154,7 +156,7 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
                             onToolbarItemSelected={this.onToolbarItemSelected} />
                     </div>
                     <div className="editor-page-content-body">
```

Comment:
Since the `AssetPreview` experience already displays an error you shouldn't need to do anything here directly in the editor page.